packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/destinationConnector-XCA Endpoint-responseTransformer-step-1-Case 1 - Success or PartialSuccess.js

```diff
@@ -1,17 +1,18 @@
-// If both successes and failures are received from Responding Gateways, the Initiating Gateway shall return both DocumentResponse and 
+// If both successes and failures are received from Responding Gateways, the Initiating Gateway shall return both DocumentResponse and
 // RegistryErrorList elements in one response and specify PartialSuccess status.
 
 if ('Success' == queryResponseCode.toString() || 'PartialSuccess' == queryResponseCode.toString()) {
 
 	if (xml.*::DocumentResponse.length() > 0) try {
 
+    var bucketName = java.lang.String(configurationMap.get('S3.BucketName'));
```

Comment:
for my own knowledge: how does mirth pick up these changes? do we have some way to load the latest code into mirth on every build?

Reply:
Locally, I've been running this script that updates the Mirth configurations: 
`./scripts/push-to-server.sh` from the `ihe-gateway` directory

Reply:
Not sure how it would be done in cloud but yea we would need to run the script Ramil suggested